thirdparty/xmlstream/xmlstream_p.h

```diff
@@ -1140,10 +1140,11 @@ bool XmlStreamReaderPrivate::parse()
             case '\n':
                 ++lineNumber;
                 lastLineStart = characterOffset + readBufferPos;
+                [[gnu::fallthrough]];
```

Comment:
Re. thirdparty: Exactly the place where the warning occurs apparently has already been modified for MuseScore when you look at the comment there.
To keep the original style maybe just mark the fallthrough with the same comment as in line 1128 ? - which as we now know at least works for gcc...